sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -223,10 +223,9 @@ public async Task AnalyzeDocumentCanParseMultipageBusinessCard(bool useStream)
         public async Task AnalyzeDocumentWithCustomModel(bool useStream)
         {
             var client = CreateDocumentAnalysisClient();
-            var modelId = Recording.GenerateId();
             AnalyzeDocumentOperation operation;
 
-            await using var customModel = await BuildDisposableDocumentModelAsync(modelId);
+            await using var customModel = await BuildDisposableDocumentModelAsync();
```

Comment:
Just placed the `Recording.GenerateId()` call inside the `BuildDisposableDocumentModelAsync` function so we don't need to invoke it every time.